charts/dependency-track/values.yaml

```diff
@@ -175,6 +176,6 @@ postgresql:
   enabled: true
   external: true #If true, disable postgresql dependency
   postgresqlUsername: deptrack
-  postgresqlPassword: deptrack
+  postgresqlPassword: ""
   postgresqlDatabase: deptrack
   postgresqlHost: dependencytrack-database-server.postgres.database.azure.com:5432
```

Comment:
provide this information also on installation. 
be aware that this repo is public.